transcode/transcode.go

```diff
@@ -85,7 +85,7 @@ func init() {
 	LocalBroadcasterClient = b
 }
 
-func RunTranscodeProcess(transcodeRequest TranscodeSegmentRequest, streamName string, inputInfo clients.InputVideo) ([]clients.OutputVideo, error) {
+func RunTranscodeProcess(transcodeRequest TranscodeSegmentRequest, streamName string, inputInfo clients.InputVideo, segmentsCounter func()) ([]clients.OutputVideo, error) {
```

Comment:
I think I prefer having it return a struct with information about the source (i.e a third return object) to passing in this counting method